analyzer/src/funTest/assets/projects/synthetic/dotnet-direct-dependencies-only-expected-output.yml

```diff
@@ -68,25 +129,20 @@ packages:
 - id: "NuGet::System.Threading:4.0.11"
   purl: "pkg:nuget/System.Threading@4.0.11"
   authors:
-  - "Microsoft"
+  - "Microsoft <>"
   declared_licenses:
-  - "http://go.microsoft.com/fwlink/?LinkId=329770"
+  - "LicenseUrl: http://go.microsoft.com/fwlink/?LinkId=329770"
```

Comment:
The declared license we return is a string encoded as YAML and is based on the actual data available upstream that can be a LicenseUrl or various LicenseMetadata (including an SPDX expression in some case) or both.